changelog/README.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1st July 2022
+
+* Added [Availability blocks](../booking-engine-api/operations/availability-blocks.md) documentation.
```

Comment:
I think all the changes should be included in the Changelog:
* Added new operation [Get availability blocks](../booking-engine-api/operations/availability-blocks.md#get-availability-blocks)
* `AvailabilityBlockId` property added to [Get availability](../booking-engine-api/operations/hotels.md#get-availability), [Create reservation groups](../booking-engine-api/operations/reservation-groups.md#create-reservation-groups) and [Get reservation pricing](../booking-engine-api/operations/reservations.md#get-reservation-pricing)
* Added new [Availability Blocks Use Case](../booking-engine-api/use-cases/availability-blocks.md)